contracts/DeveloperRegistry.sol

```diff
@@ -149,7 +165,7 @@ contract DeveloperRegistry is Ownable {
      *
      * @param _developerOwner       Address that has the ability to create a new DeveloperRegistrar with the below nameHash
      */
-    function removeAllowedDeveloper(address _developerOwner) external onlyOwner {
+    function removeAllowedDeveloper(address _developerOwner) external onlyNameGovernor {
```

Comment:
This seems correct to me, but I am trying to run through scenarios -- the main one being a compromise of the NameGovernor which then allows for removal of a bunch of legitimate TSMs.

Reply:
Never mind, I see how this is called from NameGovernor via onlyOwner, so we're fine.